### ISSUE TITLE
Impaired renal tubular reabsorption of low-molecular proteins 

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6866]
**Preferred term label:**
Impaired renal tubular reabsorption of low-molecular proteins 

**Synonyms**

**Definition (free text, please give PubMed ID)**
Decreased renal tubular reabsorption of freely filtered low-molecular proteins.
PMID: 31474092

**Parent term (use hpo.jax.org/app)**
Abnormality of renal resorption HP:0011038

**Associated term**
Renal Fanconi syndrome HP:0001994

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
@RichardKlaus  I think that the observation underlying these five terms is actually related to Increased fractional excretion of X. The HPO does not encode inference about the pathophysiology, which require clinical reasoning and context, but instead tries to remain close to the primary observations. I will add these concepts as Fractional Excretion next week, but please get back to me if you have comments/suggestions!

**Comment by pnrobinson** [comment]
@RichardKlaus  adding as follows, please reopen if inaccurate.
Increased low molecular weight protein fractional excretion

def: Increased loss of low-molecular weight proteins by the kidneys, measured in terms of the fractional excretion (FE). The FE of low-molecular-weight proteins such as beta2-microglobulin can be measured in terms of plasma and urine concentrations, since urinary low-molecular-weight protein concentrations can vary with water reabsorption. 
Comment: Urinary proteins are reabsorbed by the renal tubule cells, and increased low molecular weight protein fractional excretion can be a sign of tubular proteinuria.

**Comment by pnrobinson** [comment]
added as HP:0034356
